pkg/components/components.go

```diff
@@ -477,7 +477,7 @@ func GetClusterPermissions() []rbacv1.PolicyRule {
 		},
 		{
 			APIGroups: emptyAPIGroup,
-			Resources: stringListToSlice("nodes"),
+			Resources: stringListToSlice("nodes", "nodes/proxy"),
```

Comment:
needed to allow HCO to fetch the node configuration to know the maxImages config